tomviz/PipelineModel.cxx

```diff
@@ -182,6 +182,9 @@ bool PipelineModel::TreeItem::remove(Module *module)
   foreach(auto childItem, m_children) {
     if (childItem->module() == module) {
       removeChild(childItem->childIndex());
+      // Not sure I like this, an alternative is to make TreeItem a
+      // QObject and use a signal?
+      ModuleManager::instance().removeModule(module);
       return true;
```

Comment:
Any suggests here? I think its better than the double traversal we where doing before.